### DISCUSSION TITLE
Scar's Legion location info not showing?

### DISCUSSION THREAD

**Discussion by trashpost** [endless-sky/endless-sky#9935] [Q&A] [answered]
Started playing again. The save I'm on began this mission about ~1.5/2 years ago when it was last played.
Haihome has the prompt to search the pirate starports, but no information is given at any pirate starport. I had all the pirate planets dominated (except for the new ones) and tried freeing them, the revisiting to see if that would get things going, but no luck.

MIght just be caused by the fact the save is old, but I'm wondering if there's a fix to get the mission to progress.

**Comment by bene-dictator** [comment]
Interactions involving Scar's Legion have changed significantly in the past two years. I don't have a fix right now, though it might be as easy as adding a line that says the mission is done and going back to Hai-home; I'm not sure.

We anticipate to rewrite the missions Scar's Legion is part of in the near future, but I can't say when they will be finished.

**Reply by trashpost** [reply]
I assumed the main issue was how old the save is. I could just drop/ignore it, but I'm not sure if its part of a chain.
What would I need to change in the save file?

**Reply by MasterOfGrey** [reply, marked as answer]
Due the extent of what the changes eventually became, the original missions no longer exist. So it's part of a chain - but you're not in that chain anymore.
You can safely drop it. The current version is temporarily locked off behind a content gate awaiting resolution.
